packages/app/src/components/OutputList.vue

```diff
@@ -10,6 +10,7 @@
 import { defineComponent, onMounted, ref } from 'vue';
 import { Pet } from '@project-leek/commons';
 import { Paginated } from '@feathersjs/feathers';
+
 import feathers from '../lib/feathers';
 import entry from './Entry.vue';
```

Comment:
```suggestion
import { defineComponent, onMounted, ref } from 'vue';
import feathers from '../lib/feathers';
import { Paginated } from '@feathersjs/feathers';

import { Pet } from '@project-leek/commons';
import entry from './Entry.vue';
```

Reply:
The original one had a separation between external and internal modules.